engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/VolumeOrchestrator.java

```diff
@@ -1235,12 +1280,10 @@ public Volume migrateVolume(Volume volume, StoragePool destPool) throws StorageU
                 }
             }
             return result.getVolume();
-        } catch (InterruptedException e) {
-            s_logger.debug("migrate volume failed", e);
-            throw new CloudRuntimeException(e.getMessage());
-        } catch (ExecutionException e) {
-            s_logger.debug("migrate volume failed", e);
-            throw new CloudRuntimeException(e.getMessage());
+        } catch (InterruptedException | ExecutionException e) {
+            String msg = String.format("Volume [%s] migration failed due to [%s].", volumeToString, e.getMessage());
+            s_logger.error(msg, e);
```

Comment:
please don´t log error with exception. If need be add a debug that logs the stacktrace